### PR TITLE
PT-743: update rekisteriseloste links + fix broken test from master

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
addition to links, also fixed broken tests which may have occured after automatic merge to master.
### Closes :no_good_woman:
https://helsinkisolutionoffice.atlassian.net/browse/PT-743

## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx [hunk 1]
```diff
@@ -358,22 +358,20 @@ test('does not initializes values from URL if they are invalid', async () => {
...
       },
       currentDate,
       1
-    )
-  );
+    );
+  });
 });
 
 test('when only one group checkbox is checked, amount of seats should be disabled', async () => {
```
Comment (nenonja) [inline src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx:373]: there were some merging bugs

File: src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+exports[`Footer matches snapshot 1`] = `"<footer class=\\"footerWrapper\\"><div class=\\"Koros-module_koros__1r3rg koros\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" aria-hidden=\\"true\\" width=\\"100%\\" height=\\"85\\"><defs><pattern id=\\"koros_basic-1\\" x=\\"0\\" y=\\"0\\" width=\\"106\\" height=\\"85\\" patternUnits=\\"userSpaceOnUse\\"><path transform=\\"scale(5.3)\\" d=\\"M0,800h20V0c-4.9,0-5,2.6-9.9,2.6S5,0,0,0V800z\\"></path></pattern></defs><rect fill=\\"url(#koros_basic-1)\\" width=\\"100%\\" height=\\"85\\"></rect></svg></div><div class=\\"container default\\"><div class=\\"footer\\"><div class=\\"helsinkiLogo\\"></div><div class=\\"copyright\\"><div>© Copyright 2020 • Kaikki oikeudet pidätetään</div><div><a href=\\"https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Kuva/Henkil%C3%B6rekisteri%20Palvelutarjotin.pdf\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Rekisteriseloste</a></div></div></div></div></footer>"`;
```
Comment (Sartonon) [inline src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap:3]: Seems liek Footer snapshot has not been well formatted and now it is really hard to see changes.. Can you change that to this PR?

**Review by Sartonon — COMMENTED** [review]
Looks good and tests seems to be working. Should we still update how the snapshot is handled?

**Review by nikomakela — APPROVED** [review]

File: src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap (hunk 2 above)
Comment (nenonja) [reply src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap:3]: fixed now all enzyme snapshots to testinglibrary snapshots

**Review by Sartonon — APPROVED** [review]

**Pull request merged**